.flake8

```diff
@@ -1,8 +1,13 @@
 [flake8]
+max-line-length = 80
+select = C,E,F,W,B,B9
+ignore = E203, E501, W503, B006, E231
```

Comment:
remove `E501`; black does a great job of avoiding long lines.
`B006` mutable default args are problematic and should be fixed not ignored.
`E231` spaces after commas are nice, and black will always do that; this exclusion shouldn't be needed.

Reply:
Noted - I copied this from lando-api. Will update this here (and make a note to update in lando-api)